.github/workflows/android-build.yml

```diff
@@ -0,0 +1,54 @@
+name: Build Sample for Android
+
+on:
+  push
+
+jobs:
+  apk:
+    name: Apk Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: android-build-with-cmake

```

Comment:
As far as I know, actions/checkout@v3 takes the current branch, when used with [push]. We don't need to manually specify the ref as far as I know, instead this should take the correct branch automatically when you're testing on your own repo.